datastore/index.yaml

```diff
@@ -0,0 +1,54 @@
+indexes:
+
+- kind: Task
+  ancestor: no
+  properties:
+  - name: done
+  - name: priority
+    direction: desc
+
+- kind: Task
+  properties:
+  - name: collaborators
+    direction: asc
+  - name: created
+    direction: desc
+
+- kind: Task
+  ancestor: no
+  properties:
+  - name: priority
+    direction: asc
+  - name: created
+    direction: asc
+
+- kind: Task
+  ancestor: no
+  properties:
+  - name: done
+  - name: priority
+    direction: desc
+  - name: created
+    direction: asc
+
```

Comment:
For `multi_sort` to work, I had to add the following index:

```yaml
- kind: Task                                                                       
  ancestor: no                                                                     
  properties:                                                                      
  - name: priority                                                                 
    direction: desc                                                                
  - name: created                                                                  
    direction: asc
```

Adding this made all of my specs pass